tests/cpp-tests/Classes/AppDelegate.cpp

```diff
@@ -165,12 +165,7 @@ bool AppDelegate::applicationDidFinishLaunching()
     
     fileUtils->setSearchPaths(searchPaths);
 
-#if (CC_TARGET_PLATFORM == CC_PLATFORM_WP8)
-    // a bug in DirectX 11 level9-x on the device prevents ResolutionPolicy::NO_BORDER from working correctly
     glview->setDesignResolutionSize(designSize.width, designSize.height, ResolutionPolicy::SHOW_ALL);
```

Comment:
@stammen , do you know why it doesn't work correctly in `NO_BORDER` mode for DirectX 11 level9-x?
Doesn't it support that view port size is larger than device screen size?